snuba/schemas.py

```diff
@@ -211,10 +215,10 @@
     'properties': {
         'project': {
             'anyOf': [
-                {'type': 'integer'},
+                {'type': 'integer', 'minimum': 0},
                 {
                     'type': 'array',
-                    'items': {'type': 'integer'},
+                    'items': {'type': 'integer', 'minimum': 0},
```

Comment:
This is correct for the type, but isn't a valid project ID in practice. Might be more appropriate to make this a minimum of 1?

Reply:
I would try 1

Reply:
Sounds good, I updated this with 152c7a76510a7c167089dc4c3b3e1b1e74d901af.